src/bytes/lin_tests_dsl.ml

```diff
@@ -8,6 +8,7 @@ module BConf = struct
 
   open Lin_api
   let int = nat_small
+  let string = string_small
```

Comment:
Is this related to the rest of that commit?

Reply:
Early on I spotted that we were generating long `string`s in the `bytes` tests - worst case, up to length 10000 (arising from `Gen.nat`). The change alters that to `string`s of up to length 100 (arising from `Gen.small_nat`), which is more in line with our other tests that generate `string`s.